src/util.rs

```diff
@@ -4,6 +4,16 @@ use std::io::Read;
 use sha2::{Sha256, Digest};
 use tar::Archive;
 
+pub fn get_efi_mnt() -> Option<&'static str> {
+   let efi = path::Path::new("EFI");
```

Comment:
This is pretty good. A potentially better way to do this might be to get the output of `bootctl --print-esp-path`

Reply:
But this is acceptable to me for now

Reply:
Thanks for the suggestion, I added a commit to try `bootctl --print-esp-path` first and fallback to the hard-coded /boot or /boot/efi location test.